test/integration/job/job_test.go

```diff
@@ -3477,8 +3525,8 @@ func BenchmarkLargeFailureHandling(b *testing.B) {
 	}
 }
 
-// cleanUp deletes all pods and the job
-func cleanUp(ctx context.Context, clientSet clientset.Interface, jobObj *batchv1.Job) error {
+// cleanUp deletes all pods and the job. Waits for the job deletion if sync is true
```

Comment:
```suggestion
// cleanUp deletes all pods and the job and waits for the job deletion if sync is set to true.
```

Reply:
mm and what do you think of

```suggestion
// cleanUp deletes all pods and the job, then waits for job deletion if sync is set to true.
```